docs/flow/modin/core/execution/unidist/generic.rst

```diff
@@ -0,0 +1,19 @@
+:orphan:
+
+Generic Unidist-based members
+=============================
+
+Objects which are storage format agnostic but require specific Unidist implementation
+are placed in ``modin.core.execution.unidist.generic``.
+
+Their purpose is to implement certain parallel I/O operations and to serve
+as a foundation for building storage format specific objects:
+
+* :py:class:`~modin.core.execution.unidist.generic.io.UnidistIO` -- implements parallel :meth:`~modin.core.execution.unidist.generic.io.UnidistIO.to_csv` and :meth:`~modin.core.execution.unidist.generic.io.UnidistIO.to_sql`.
```

Comment:
```suggestion
```
This should be removed because `to_csv` and `to_sql` are implemented in PandasOnUnidistIO. We should fix that for Ray engine too as part of a separate issue. Please create one.